src/server-extension/helper.ts

```
@@ -14,3 +14,8 @@ export const getProjectSortBy = (sortBy: EProjectSort) => {
       return { sortBy: "vouch", order: "DESC" };
   }
 };
+
+export function isValidDate(dateStr: string): boolean {
+  const date = new Date(dateStr);
+  return !isNaN(date.getTime()); // Check if the date is valid
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)